css/vendor-imports/mozilla/mozilla-central-reftests/selectors4/focus-within-1.html

```diff
@@ -8,7 +8,6 @@
     <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
     <meta name="assert" content="Test checks :focus-within works after focus change">
     <link rel="match" href="focus-within-1-ref.html">
-    <meta name="flags" content="interact">
```

Comment:
Similarly to focus-within-001:

*  It is better to switch to `<meta name="flags" content="">` than to remove the line. Makes it clear that you have considered the question and decided that there was no need for flags, instead of maybe forgetting to think about it.

* As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.